README.md

```diff
@@ -2,28 +2,20 @@
 
 # Parity Fether - a fast and decentralized wallet based on a light-client
 
-## Get the latest binary
+## [» Download the latest release «](https://github.com/paritytech/fether/releases)
 
-[» Download the latest release «](https://github.com/paritytech/fether/releases)
-
-### Join the chat!
-
-Get in touch with us on Gitter:
-[![Gitter](https://img.shields.io/badge/Gitter-Fether-brightgreen.svg)](https://gitter.im/paritytech/fether)
-
-Official website: https://parity.io | Be sure to check out [our Wiki](https://wiki.parity.io) for more information.
 
 ---
 
 ## About Parity Fether
 
-Fether aims to be the lightest and simplest decentralized wallet. It supports Ether and ERC-20 tokens and runs on top of [Parity Ethereum](https://github.com/paritytech/parity-ethereum) light client allowing smooth synchronization and interaction with the Ethereum blockchain, in a decentralized manner.
+Parity Fether aims to be the lightest and simplest decentralized wallet. It supports Ether and ERC-20 tokens, and runs on top of [Parity Ethereum](https://github.com/paritytech/parity-ethereum) light client. This allows smooth synchronization and interaction with the Ethereum blockchain, in a decentralized manner.
 
 By default, Parity Fether alpha runs on the Kovan test network. You can receive free Kovan Ether by posting your address in the [Kovan Faucet](https://gitter.im/kovan-testnet/faucet) Gitter channel. Fether will download and launch Parity Ethereum node at startup if it's not found on the computer. You can also separately launch your Ethereum client, Fether will automatically connect to it.
 
-If you run into problems while using Parity Fether, feel free to file an issue in this repository or hop on our [Gitter](https://gitter.im/paritytech/parity) or [Riot](https://riot.im/app/#/group/+parity:matrix.parity.io) chat room to ask a question. We are glad to help! **For security-critical issues**, please refer to the security policy outlined in [SECURITY.md](https://github.com/paritytech/parity/blob/master/SECURITY.md).
+Parity Fether connects to the light node using [`@parity/light.js`](https://github.com/paritytech/js-libs/tree/master/packages/light.js), a Javascript library specifically crafted for wallets to connect with light clients. It is licensed under the BSD 3-Clause, and can be used for all your Ethereum needs.
```

Comment:
Just saw this: light.js is licensed under MIT. Whom does "it" refer to?

Reply:
Good point, "it" used to be for Fether, and it should remain so.